### PR TITLE
Test test ❌

### PR DESCRIPTION
**Pull request by DanGould**
#13 passed CI when it should not have, because the loin server failed to connect. However, it passes with a successful payjoin on my machine. I'm pushing this to run CI and see if I can replicate the problem.
https://github.com/chaincase-app/nolooking/actions/runs/3325555814
/jobs/5498363364

### CODE REVIEW

**Comment by DanGould** [comment]
Aha, the test passes despite a panicked thread. Two things need to be done:

1. make sure if critical threads panic the test fails. `assert!(has_opened_channel)`
2. loop until server@3000 is accessible from 3010. I pj panics because of a race. with the 2 second timeout

**Comment by DanGould** [comment]
oh.. I accidentally killed the nginx `events {}` section which invalidated the configuration.

I missed it because the test was passing regardless of a channel open

File: .github/workflows/nginx/ssl/.placeholder [hunk 1]
```diff
@@ -0,0 +1 @@
+# this file keeps the directory around for tests
```
Comment (nickfarrow) [inline .github/workflows/nginx/ssl/.placeholder:1]: Can this be an empty `.gitignore`?

**Comment by nickfarrow** [comment]
yeah you knew it, damn docker L179 integration.rs needs 172.17.0.1.

Also maybe cherry-pick /pull this commit onto your test-test branch, makes tests fail depending on test success.
https://github.com/nickfarrow/loin/commit/f97d3ce45b4ad81735aeb02a1e476e2270f82958
If the channel opens, we set the Fixture to have `test_succeedeed==true`, which we later use in `drop()` to maybe panic.

**Comment by DanGould** [comment]
@nickfarrow Huzzah!!! you fixed it! This one was a headache to find. 

Binding to 172.17.0.2 doesn't run on my mac however, so we need a dynamic var to bind to.

**Comment by nickfarrow** [comment]
Nice! LGTM

File: .github/workflows/nginx/ssl/.placeholder (hunk 1 above)
Comment (DanGould) [reply .github/workflows/nginx/ssl/.placeholder]: I force-pushed a .gitignore. The rest is the same

**Review by nickfarrow — APPROVED** [review]

**Pull request merged**